src/lib/specs-crawler.js

```diff
@@ -392,6 +403,7 @@ async function crawlList(speclist, crawlOptions) {
         const spec = specAndPromise.spec;
         const logCounter = ('' + (idx + 1)).padStart(nbStr.length, ' ') + '/' + nbStr;
         crawlOptions.quiet ?? console.warn(`${logCounter} - ${spec.url} - crawling`);
+        crawlOptions.logCounter = logCounter;
```

Comment:
I'm surprised that this works given that several specs are crawled in parallel. You'll need to make a shallow copy of `crawlOptions` or use another mechanism to pass that parameter.

Reply:
it doesn't work, and was removed from my local copy but I failed to update the committed code :)